dnsapi/dns_ispconfig.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env sh
+
+#ISPConfig 3.1 API - Add remote user and give him access to at least the "DNS txt functions"
+
```

Comment:
Please add your github link here, so that if there is any bug found,  the user know where to report bug.